app/models/member.rb

```diff
@@ -8,7 +8,8 @@ class Member < ActiveRecord::Base
   def self.find_from_request(akid: nil, id: nil)
     if akid.present?
       actionkit_user_id = AkidParser.parse(akid)[:actionkit_user_id]
-      member = find_by(actionkit_user_id: actionkit_user_id)
+      #member = find_by(actionkit_user_id: actionkit_user_id)
+      member = where(actionkit_user_id: actionkit_user_id).order('created_at ASC').first
```

Comment:
Pretty curious why `where` is better than `find_by` - care to let us in on the performance tip?